Algorithmia/CLI.py

```diff
@@ -0,0 +1,233 @@
+import Algorithmia
+import os
+from Algorithmia.algo_response import AlgoResponse
+import json, re, requests
+
+class CLI():
+	def __init__(self):
+		self.client = Algorithmia.client()
+		# algo auth
+	def auth(self, apikey):
+
+		#setenv = "export ALGORITHMIA_API_KEY=" + "'" + APIkey + "'"
+		#print(setenv)
+
+		#store api key in local config file and read from it each time a client needs to be created
+		key = open(os.environ['HOME']+"/.algorithmia_api_key","w")
+		key.write(apikey)
+		key.close()
+
+		print("apikey is:")
+		key = open(os.environ['HOME']+"/.algorithmia_api_key","r")
+		print(key.read())
+		key.close()
+
+	# algo run <algo> <args..>    run the the spesified algo
+	def runalgo(self, name, inputs, client):
+		algo_name = name
+
+		algo_input = inputs[0]
+
+		algo = client.algo(algo_name)
+		
+		result = None
+		'''
+		time = 300
+		if("--timeout" in inputs):
+			#find timeout value
```

Comment:
is this a dev facing comment or something you aren't using and removed? In any case this is the wrong place for it.